moviebot/recommender/slot_based_recommender_model.py

```diff
@@ -0,0 +1,35 @@
+"""Recommender model based on slot value pairs."""
+
+from typing import Any, Dict, List
+
+from moviebot.database.database import DataBase
+from moviebot.dialogue_manager.dialogue_state import DialogueState
+from moviebot.ontology.ontology import Ontology
+from moviebot.recommender.recommender_model import RecommenderModel
+
+
+class SlotBasedRecommenderModel(RecommenderModel):
+    def __init__(self, db: DataBase, ontology: Ontology) -> None:
```

Comment:
Do we actually need here __init__ method? It is the same as in `RecommenderModel` class.

Reply:
Good point! I removed it.